src/BPEssentials/Commands/Moderation/Warns/WarnRemove.cs

```diff
@@ -0,0 +1,49 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BPEssentials.ExtensionMethods.Warns;
+using BrokeProtocol.Collections;
+using BrokeProtocol.Entities;
+using System.Collections.Generic;
+using static BPEssentials.ExtensionMethods.Warns.ExtensionPlayerWarns;
+
+namespace BPEssentials.Commands
+{
+    public class WarnRemove : Command
+    {
+        public void Invoke(ShPlayer player, string target, int warnId)
+        {
+            if (warnId < 1)
+            {
+                player.TS("warn_remove_error_null_or_negative", warnId.ToString());
+                return;
+            }
+
+            if (EntityCollections.TryGetPlayerByNameOrID(target, out var shTarget))
+            {
+                if (CheckWarnCount(player, warnId, shTarget.GetWarns())) { return; }
```

Comment:
Place return statements on new line.
```suggestion
                if (CheckWarnCount(player, warnId, shTarget.GetWarns()))
                {
                    return;
                }
```